### PR TITLE
Feature/ge 813

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: authentication_service/context_processors.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    session_client_name = request.session.get(
+        EXTRA_SESSION_KEY,
+        {}
+    ).get(COOKIES["redirect_client_name"])
+    return {
+        "ge_global_redirect_uri": request.COOKIES.get(
+            COOKIES["redirect_cookie"]
+        ),
+        "ge_global_client_name": request.COOKIES.get(
+            COOKIES["redirect_client_name"], session_client_name
```
Comment (cobusc) [inline authentication_service/context_processors.py]: Is this the client name or client id?

Comment (AltusBarry) [reply authentication_service/context_processors.py]: Name. Its to be displayed on a template.

File: authentication_service/middleware.py [hunk 2]
```diff
@@ -2,8 +2,19 @@
 import logging
 import os
 
+from oidc_provider.lib.endpoints.authorize import AuthorizeEndpoint
+from oidc_provider.lib.errors import (
```
Comment (ruands) [inline authentication_service/middleware.py]: Didn't know you could import like this. Pretty neat.

File: authentication_service/context_processors.py (hunk 1 above)
Comment (cobusc) [reply authentication_service/context_processors.py]: If that is the case then this cookie is delicious.

**Comment by cobusc** [comment]
:+1:

**Pull request closed without merging**